### DISCUSSION TITLE
No transmitter mac address xDrip+ android

### DISCUSSION THREAD

**Discussion by skh1990** [NightscoutFoundation/xDrip#2538] [Q&A] [answered]
![Screenshot_20221207-013604](https://user-images.githubusercontent.com/56776478/206210381-3efea366-607e-49e2-b402-64c8d8ba0167.png)

This is all I'm getting when I try to pair my transmitter. This has never happened before, I'm at a loss. I uninstalled and reinstalled to see if that would help, I have looked and made sure all my settings are right, that didn't work. I've played around with some suggestions from others on Facebook and that didn't work. This is a first, I'm hoping someone here knows the answer. Or maybe this transmitter is just a dud.

**Comment by Navid200** [comment, marked as answer]
Where is the transmitter?
A new transmitter needs to be activated.  That happens when you insert it in a sensor on your body.
You can also activate if you hold it in your hand with your thumb pressed against the two contacts for a few minutes.

But, only activate if you intend to use.  You will not be able to deactivate after.

**Reply by skh1990** [reply]
Hey Navid, this is a brand new transmitter that I'm trying to pair after my last one just died. It is not on a sensor yet as I haven't been able to get it to pair and show me battery levels. It is out of the box and sitting beside my phone. (I think you just replied to me on Facebook as well, I've been at this all night and so exhausted with this thing)

**Comment by skh1990** [comment]
I just held it with my thumb like you said and in less than a minute it gave me a pair request, halle fricken lujah! Thank you, who would have thought it would be so simple 😅😅

**Reply by Navid200** [reply]
Great.  Sorry it frustrated you

**Reply by skh1990** [reply]
I will remember that tip if it ever happens again, I was about to throw it away 😳 thank you so much
